nexus-sdk-js/src/Resource/index.ts

```diff
@@ -121,15 +121,15 @@ export default class Resource<T = {}> {
     context,
     ...data
   }: {
-    context: { [field: string]: string };
+    context?: Context;
     [field: string]: any;
   }): Promise<Resource> {
     return Resource.updateSelf(
       this.resourceURL,
       this.rev,
       {
         context,
-        data,
+        ...data,
```

Comment:
if this didn't break any tests, that means it isn't tested!

Reply:
I've added a nice test